cldr/README

```diff
@@ -0,0 +1,36 @@
+The JSON files published here are provided as a reference sample to show the recommended structure for JSON
```

Comment:
These files are used pretty widely (e.g. jQuery's https://github.com/rxaviers/cldrjs) and are considered the "official" JSON version. Other might be able to build them also, but few do outside of some older libraries that got started before these were first distributed in 2013.  More color at this talk by the jQuery Globalize lead: http://rafael.xavier.blog.br/2014/12/23/a-story-of-jquery-cldr-json-and-the-community-at-the-unicode-santa-clara-ca/